ddcz/views/postal_service.py

```diff
@@ -20,60 +20,31 @@
 FORM_SEND = 2
 FORM_REPLY = 4
 
+DEFAULT_LIMIT = 10
+DEFAULT_PAGE = 1
+
 logger = logging.getLogger(__name__)
 
 
 @login_required
 @require_http_methods(["HEAD", "GET", "POST"])
 def postal_service(request):
     if request.method == "POST":
-
-        fid = int(request.POST.get("fid"))
-        if fid == FORM_SEND:
-
-            try:
-                Letters.objects.create(
-                    receiver=UserProfile.objects.get(id=request.POST.get("whom")).nick,
-                    sender=request.user.userprofile.nick,
-                    text=request.POST.get("text"),
-                    date=datetime.now(),
-                    visibility=1,
-                )
-                return HttpResponseRedirect(reverse("ddcz:postal-service"))
-
-            except UserProfile.DoesNotExist:
-                id = request.POST.get("whom")
-                logger.error(
-                    f"A message between user has been submitted but the receiver of userprofile can not be found in our database. ID: {id}"
-                )
-
-        elif fid == FORM_DELETE:
-
-            try:
-                letter = Letters.objects.filter(pk=request.POST.get("id", 0)).update(
-                    visibility=0
-                )
-                return HttpResponseRedirect(reverse("ddcz:postal-service"))
-
-            except Letters.DoesNotExist:
-                id = request.POST.get("id")
-                user = request.user.userprofile.nick
-                logger.error(
-                    f"There has been an attempt to delete a message with non existing id: {id}, User: {user}"
-                )
-                return HttpResponseRedirect(reverse("ddcz:postal-service"))
+        return handle_postal_service_post_request(request)
 
     nick = request.user.userprofile.nick
-    limit = int(request.GET.get("l", 50))
-    page = int(request.GET.get("p", 1))
+    per_page = int(request.GET.get("l", DEFAULT_LIMIT))
+    page = int(request.GET.get("z_s", DEFAULT_PAGE))
+
     letters = Letters.objects.filter(
         (Q(receiver=nick) | Q(sender=nick)) & Q(visibility=1)
     ).order_by("-date")
+
     box_occupancy = Letters.objects.filter(
         (Q(receiver=nick) | Q(sender=nick)) & Q(visibility=1)
     ).count()
 
-    paginator = Paginator(letters, limit)
+    paginator = Paginator(letters, per_page)
     letters = paginator.get_page(page)
 
     return render(
```

Comment:
I feel like I am missing a PR somewhere since I don't see the `postal_office` template PR 🤔